src/NuGet.Clients/NuGet.PackageManagement.UI/Options/GeneralOptionControl.cs

```diff
@@ -59,14 +59,21 @@ internal void OnActivated()
                     showPackageManagementChooser.Checked = packageManagement.Enabled;
 #endif
                 }
-                catch (InvalidOperationException)
+                // Throwed during creating or saving NuGet.Config.
+                catch (NuGetConfigurationException ex)
                 {
-                    MessageHelper.ShowErrorMessage(Resources.ShowError_ConfigInvalidOperation, Resources.ErrorDialogBoxTitle);
-                } 
+                    MessageHelper.ShowErrorMessage(ex.Message, Resources.ErrorDialogBoxTitle);
+                }
+                // Throwed if no nuget.config found.
+                catch (InvalidOperationException ex)
+                {
+                    MessageHelper.ShowErrorMessage(ex.Message, Resources.ErrorDialogBoxTitle);
```

Comment:
Shouldn't this be Resources.ShowError_ConfigInvalidOperation?

Reply:
we already have the message from Settings